File: src/main/java/com/temelyan/pomoapp/WebApplication.java
package com.temelyan.pomoapp;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cache.annotation.EnableCaching;

@SpringBootApplication
@EnableCaching
public class WebApplication {

    public static void main(String[] args) {
        new SpringApplication(WebApplication.class).run(args);
    }
}

File: src/main/java/com/temelyan/pomoapp/WebApplication.kt
package com.temelyan.pomoapp

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cache.annotation.EnableCaching

@SpringBootApplication
@EnableCaching
class WebApplication

fun main(args: Array<String>) {
    runApplication<WebApplication>(*args)
}

